src/main/java/uk/gov/pay/api/app/config/PublicApiModule.java

```diff
@@ -79,7 +78,16 @@ public RateLimiterConfig getRateLimiterConfig() {
     @Singleton
     public RedisClient getRedisClient() {
         RedisClient client = RedisClient.create(configuration.getRedisConfiguration().getUrl());
-        client.setDefaultTimeout(Duration.ofMillis(configuration.getRedisConfiguration().getTimeout()));
+        SocketOptions socketOptions = SocketOptions
+                .builder()
+                .connectTimeout(Duration.ofMillis(configuration.getRedisConfiguration().getSocketTimeout()))
```

Comment:
Can the config parameter be named `CONNECT_TIMEOUT` please rather than `SOCKET_TIMEOUT` because we are setting the `connectTimeout` (and then update the config method names as appropriate e.g. `s/getSocketTimeout/getConnectTimeout/`). It will then also align with the Lettuce client variable names and be easier to find and understand in future: https://lettuce.io/core/release/api/io/lettuce/core/SocketOptions.html#DEFAULT_CONNECT_TIMEOUT

Reply:
Done.